### PR TITLE
Fix shield icons in Letsencrypt readme

### PR DESCRIPTION
**Pull request by ckesc**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ckesc, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#1335).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: letsencrypt/README.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ The generated certificate can be used within others addons. By default the path
 [issue]: https://github.com/home-assistant/hassio-addons/issues
 [certbot]: https://certbot.eff.org
 [reddit]: https://reddit.com/r/homeassistant
+
+[Almond]: https://almond.stanford.edu/
```
Comment (frenck) [inline letsencrypt/README.md:18]: This reference isn't used?

File: letsencrypt/README.md [hunk 2]
```diff
@@ -14,3 +14,10 @@ The generated certificate can be used within others addons. By default the path
...
+[aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
+[amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
+[armhf-shield]: https://img.shields.io/badge/armhf-no-red.svg
+[armv7-shield]: https://img.shields.io/badge/armv7-yes-green.svg
+[i386-shield]: https://img.shields.io/badge/i386-no-red.svg
```
Comment (frenck) [inline letsencrypt/README.md:23]: I believe this supports armhf and i386...

Comment (ckesc) [reply letsencrypt/README.md:22]: This ref seems to be used. You mean we should merge to shields?

**Comment by ckesc** [comment]
@frenck Fixed: I removed unused ref's

File: letsencrypt/README.md (hunk 2 above)
Comment (frenck) [reply letsencrypt/README.md:22]: I mean that for armhf and i386 it now shows a red badge with "no" in it, indicating it is not supported, while this add-on does support it.

**Comment by pvizeli** [comment]
all are yes and green

**Pull request merged**